// @sap/xssec/src/error/index.js
module.exports = {
    ConfigurationError: require('./configuration/ConfigurationError'),
    ExpiredTokenError: require('./validation/ExpiredTokenError'),
    InvalidClientCertificateError: require('./validation/InvalidClientCertificateError'),
    InvalidCredentialsError: require('./configuration/InvalidCredentialsError'),
    InvalidIssuerError: require('./validation/InvalidIssuerError'),
    InvalidJwtError: require('./validation/InvalidJwtError'),
    InvalidTokenSignatureError: require('./validation/InvalidTokenSignatureError'),
    MissingClientCertificateError: require('./validation/MissingClientCertificateError'),
    MissingIssuerError: require('./validation/MissingIssuerError'),
    MissingJwtError: require('./configuration/MissingJwtError'),
    MissingKidError: require('./validation/MissingKidError'),
    MissingVerificationKeyError: require('./validation/MissingVerificationKeyError'),
    NetworkError: require('./network/NetworkError'),
    NotYetValidTokenError: require('./validation/NotYetValidTokenError'),
    RequestError: require('./network/RequestError'),
    ResponseError: require('./network/ResponseError'),
    TokenValidationError: require('./validation/TokenValidationError'),
    UnsupportedAlgorithmError: require('./validation/UnsupportedAlgorithmError'),
    UntrustedIssuerError: require('./validation/UntrustedIssuerError'),
    ValidationError: require('./validation/ValidationError'),
    WrongAudienceError: require('./validation/WrongAudienceError'),
    X5tError: require('./validation/X5tError'),
    XssecError: require('./XssecError'),
}
